stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>Jennifer Ding:</p><ul><li>In cell 5, it might be good to apply the .describe() to just the Ref_ix + element columns, like you do in cell 11 because there is no semantic meaning to get stats on the ID/Glass Type columns (I believe?)</li></ul><p>Joe Cerniglia:</p><p>I would like to show all the columns of the pristine dataset at the outset, even ones with no semantic value, so that readers get a sense of what the original data looks like.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>